curator.gemspec

```diff
@@ -43,7 +43,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'http', '~> 4.4'
   spec.add_dependency 'oj', '~> 3.10'
   spec.add_dependency 'ox', '~> 2.13'
-  spec.add_dependency 'paper_trail', '~> 10.3' # have not implemented this yet but does version control
+  spec.add_dependency 'paper_trail', '~> 11.1'
+  spec.add_dependency 'paper_trail-association_tracking'
```

Comment:
I would add a version cap to the association_tracking in case there are any releases in the future.

Reply:
Good call, will do.